### ISSUE TITLE
netmask selection in nmap

### ISSUE THREAD

**Issue by EclipseSpark** [LionSec/xerosploit#3]
As i have added in my pull request you have hardcoded the /24 network mask

> 			# TODO:
			# the software should ask for the netmask to scan to not limit the
			# scanned hosts to 255.255.255.0
			scan = os.popen("nmap " + gateway + "/24 -n -sP ").read()

Is it possible to integrate a netmask selector to enable users to select the right ip ranges to scan?

We all use 192.168.0.0/24 or 192.168.1.0/24 at home, but most of the networks a pentester scans have custom ranges, so a range/netmak selector would be very useful

**Comment by LionSec** [comment]
Yes , you can config that by changing your gateway IP Address . When you run xerosploit , run the "gateway" command , and you could change that . 
https://youtu.be/35QUrtZEV9U?t=5m18s
I don't know this will solve your problem . If no , please give me more information . Regards .
